homeassistant/components/zwave/__init__.py

```diff
@@ -916,3 +921,15 @@ def refresh_from_network(self):
             return
         for value_id in dependent_ids + [self._value.value_id]:
             self._value.node.refresh_value(value_id)
+
+    def _schedule_update(self):
```

Comment:
We should actually make this a callback and check at start if function if an update has already been scheduled. Otherwise because of race conditions we might schedule it twice.